src/App.js

```diff
@@ -4,7 +4,13 @@ import EditNoteModal from "./Components/UI/Modal/EditNoteModal";
 import LeftNavBar from "./Components/UI/Navigation/LeftNavBar";
```

Comment:
The sign in modal was oversizing on mobile screen couldn't get pic as after sign didn,t get signout option

Reply:
Ok, will fix this.